### ISSUE TITLE
badges are encouraging counter productive behavior 

### ISSUE THREAD

**Issue by paigeadelethompson** [drknzz/GitHub-Achievements#440]
They should get rid of them. All its doing is encouraging people to chase badges and that's not what most people want to use GH for.

**Comment by EnochOluwadamilare** [comment]
Although I understand your point, I think its as well a useful way to demonstrate your impact and activity over the years. Let it stay.

**Comment by paigeadelethompson** [comment]
yeah Im sorry I was in kind of a bad mood when I wrote this. I don't think anybody would really get behind this idea anyway but I figured it ight be worth pointing out. To your point it is in theory a useful way to measure somebody's productivity but the incentive is perhaps another reason why people might abuse it. 

I'd like to take back my previous statement that it should be shelved, though because I'm not really trying to be a stick in the mud

**Comment by EnochOluwadamilare** [comment]
Exactly... Rightly stated.

**Comment by theepicsaxguy** [comment]
> yeah Im sorry I was in kind of a bad mood when I wrote this. I don't think anybody would really get behind this idea anyway but I figured it ight be worth pointing out. To your point it is in theory a useful way to measure somebody's productivity but the incentive is perhaps another reason why people might abuse it.
> 
> I'd like to take back my previous statement that it should be shelved, though because I'm not really trying to be a stick in the mud

I want to commend you for your willingness to acknowledge mistakes. It's a commendable quality, and we need more people like you.

**Comment by EnochOluwadamilare** [comment]
Very true @theepicsaxguy.

**Comment by rubydamodar** [comment]
![Doubtful & Scared GIF](https://i.pinimg.com/originals/e1/a0/e4/e1a0e460965bad32d6af9ab68ac3892e.gif)

**Comment by Adilkhan01V** [comment]
got a great point

**Comment by RagexPrince683** [comment]
I agree with both of you these both make sense. Obviously you're going to have bad actors but it can also be used as a good thing.

**Comment by EnochOluwadamilare** [comment]
Yea. 👍

**Comment by projan111** [comment]
Whatever it is, everyone would love to get achievements 🏆🏆

**Comment by sardar-mukhshaf** [comment]
getting achievements is fun, but it have to be some related to code perks as well

**Comment by MomenElsayedDev** [comment]
got a great point
